acrobat/blocks/eventwrapper/eventwrapper.js

```diff
@@ -14,6 +14,7 @@ const PREVIEW_DIS = 'preview-displayed';
 const TRY_ANOTHER = 'try-another-file-start';
 // const UPSELL_DIS = 'upsell-displayed';
 const FADE = 'review fade-in';
+const DISABLE_EXTENSION_BANNER = window.dc_hosted.disableExtensionBanner === true;
```

Comment:
can we check the flag inside the event handler? That would give dc_hosted more flexibility to set the flag based on floodgate later in the workflow. Here it might assume that the flag is set immediately on load.

Reply:
@peller Thank you. Can you please check my fix?
I used **window.dc_hosted.disableExtensionBanner !== true** rather than !window.dc_hosted.disableExtensionBanner to prevent any issues in case the value is not a boolean or undefined...